staging/src/k8s.io/apiserver/pkg/endpoints/audit_test.go

```diff
@@ -131,6 +140,7 @@ func TestAudit(t *testing.T) {
 			[]eventCheck{
 				noRequestBody(0),
 				responseBodyMatches(0, `{.*"name":"c".*}`),
+				requestUserAgentMatches(0, userAgent),
```

Comment:
Since this is constant for all testcases, just check it in the loop below rather than specifying here

Reply:
ok